tools/cbmc/proofs/HTTP/IotHttpsClient_Disconnect/IotHttpsClient_Disconnect_harness.c

```diff
@@ -0,0 +1,23 @@
+/* FreeRTOS includes. */
+#include "FreeRTOS.h"
+#include "queue.h"
+#include "FreeRTOS_Sockets.h"
+
+/* FreeRTOS+TCP includes. */
+#include "iot_https_client.h"
+#include "iot_https_internal.h"
+
+#include "../global_state_HTTP.c"
+
+void harness() {
+  IotHttpsConnectionHandle_t connHandle = newIotConnectionHandle();
+  if(connHandle) {
+  /* This is done to insert an element in reqQ and get coverage in IotHttpsClient_Disconnect. */
+  	_httpsRequest_t * pReq = malloc(sizeof(_httpsRequest_t));
+  	pReq->pHttpsResponse = malloc(sizeof(_httpsResponse_t));
+  	IotLink_t * pLink = pReq;
+  /* This function inserts the element in the reqQ.*/
+  	IotDeQueue_EnqueueTail(&(connHandle->reqQ), pLink);
```

Comment:
I think it would be more appropriate to delete line 18.
Then change line 20 to:
`IotDeQueue_EnqueueTail(&(connHandle->reqQ), &(pReq->link));`

Reply:
Done.